<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="dv-lt-n-20334" name="BankingApp Test" tests="7" failures="4" timestamp="24 Jul 2021 09:26:40 GMT" time="78.328" errors="0">
  <testcase name="loginTest" time="2.337" classname="com.inetbanking.testCases.TC_LoginTest_001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.inetbanking.testCases.TC_LoginTest_001.loginTest(TC_LoginTest_001.java:35)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginDDT" time="12.426" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="6.397" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertTrue(Assert.java:27)
	at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:32)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="6.71" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertTrue(Assert.java:27)
	at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:32)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="13.567" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="6.447" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertTrue(Assert.java:27)
	at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:32)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="addNewCustomer" time="16.445" classname="com.inetbanking.testCases.TC_AddCustomerTest_003"/>
</testsuite> <!-- BankingApp Test -->
